pkg/operator/ceph/csi/spec.go

```diff
@@ -189,7 +189,7 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		return errors.Wrapf(err, "failed creating csi config map")
 	}
 	ownerRef := metav1.OwnerReference{
-		APIVersion: "v1",
+		APIVersion: "apps/v1",
```

Comment:
Aren't ConfigMaps part of the `core/v1` API? https://github.com/kubernetes/client-go/blob/b7e8408f656ea5ae6b07f182070b2b4630f00f50/kubernetes/typed/core/v1/core_client.go#L56

Reply:
good catch! /facepalm